src/dqagi.c

```diff
@@ -72,7 +72,6 @@ double dqagi(dq_function_type f,double bound,int inf,double epsabs,
     result = G_K15I(f,boun,inf,0.0,1.0,abserr,&defabs,&resabs, user_data);
 
 /* Test on accuracy. */
-    last = 0;
```

Comment:
You are right, but this is no problem:
In dqagi.c the goto you mentioned is _after_ the variable last was set to 0. However, it is only important what happens between the first assignment (line 56) and the second assignment (line 74) which this commit removes. Between the first and second assignment the variable last is neither read nor written, and there are no jumps. (There is one return, but then we leave the whole function.) As there are no reads or writes, the variable last is still 0 in line 56, so we can remove the second assignment.